### PR TITLE
NAS-102085: Rewrite Replication Wizard documentation

### PR DESCRIPTION
**Pull request by Mrt134**
- Update screenshots to match updated wizard appearance.
- Rewrite all text to match wizard behavior and draft help text.
- Register new replacements in conf.py for generic UI buttons
- HTML build test: no issues.

### CODE REVIEW

File: userguide/tasks.rst [hunk 1]
```diff
@@ -856,109 +856,97 @@ Replication Creation Wizard
...
+ * *Never Delete*: never delete snapshots from the destination system.
 
-For example, :guilabel:`Source Dataset` :file:`storage/source1` has two
-child datasets: :file:`storage/source1/data1` and
-:file:`storage/source1/data2`. To include :file:`/data1` and
-:file:`/data2` snapshots in the replication, set :guilabel:`Recursive`.
-To keep :file:`/data1` in the replication while excluding :file:`/data2`,
-enter :literal:`storage/source1/data2` in
-:guilabel:`Exclude Child Datasets`.
+ * *Custom*: define a how long a snapshot remains on the destination
```
Comment (a-stjohn) [inline userguide/tasks.rst]:
```suggestion
 * *Custom*: define how long a snapshot remains on the destination
```

File: userguide/tasks.rst [hunk 2]
```diff
@@ -849,109 +849,97 @@ Replication Creation Wizard
...
+*On a Different System*. Use the dataset browser to select each dataset
+to include in the replication task. There is also a
+:guilabel:`Recursive` option to include child datasets with the selected
+datasets.
+
+Source datasets on a remote system need a configured
+:ref:`Periodic Snapshot task <Periodic Snapshot Tasks>` or the dataset
+snapshots must be manually selected by setting
+:guilabel:`Replicate Custom Snapshots` and entering a snapshot
+:guilabel:`Naming Schema`. The schema is the name and
```
Comment (wblock) [inline userguide/tasks.rst]:
```suggestion
:guilabel:`Naming Schema`. The schema is a pattern of the name and
```

File: userguide/tasks.rst [hunk 3]
```diff
@@ -849,109 +849,97 @@ Replication Creation Wizard
...
+`strftime(3) <https://www.freebsd.org/cgi/man.cgi?query=strftime>`__
+*%Y*, *%m*, *%d*, *%H*, and *%M* strings that match the snapshots to
```
Comment (wblock) [inline userguide/tasks.rst]:
```suggestion
*%Y*, *%m*, *%d*, *%H*, and *%M* strings that match the names of the snapshots to
```

File: userguide/tasks.rst [hunk 4]
```diff
@@ -849,109 +849,97 @@ Replication Creation Wizard
...
+include in the replication. The number of matching snapshots is shown.
+
+Now configure the :guilabel:`Destination Dataset`. Define the location
```
Comment (wblock) [inline userguide/tasks.rst]:
```suggestion
Now choose the :guilabel:`Destination Dataset` to receive the replicated snapshots.
```

File: userguide/tasks.rst [hunk 5]
```diff
@@ -849,109 +849,97 @@ Replication Creation Wizard
...
+of the dataset and use the dataset browser to select the storage
```
Comment (wblock) [inline userguide/tasks.rst]:
```suggestion
```

File: userguide/tasks.rst [hunk 6]
```diff
@@ -849,109 +849,97 @@ Replication Creation Wizard
...
+location for replicated snapshots.
```
Comment (wblock) [inline userguide/tasks.rst]:
```suggestion
```

File: userguide/tasks.rst [hunk 7]
```diff
@@ -849,109 +849,97 @@ Replication Creation Wizard
...
+
+Using an SSH connection for replication adds the
+:guilabel:`SSH Transfer Security` option. This sets the data transfer
+security level. The connection is authenticated with SSH, then data can
+either be encrypted during transfer to maximize protection or left
+unencrypted to maximize transfer speed. **WARNING:** Transferring data
+without encryption is only recommended for secure networks.
```
Comment (wblock) [inline userguide/tasks.rst]: We flipped this warning around in the tooltip, I think.

File: userguide/tasks.rst [hunk 8]
```diff
@@ -849,109 +849,97 @@ Replication Creation Wizard
...
+
+A suggested replication :guilabel:`Task Name` is shown. This can be
+overwritten, but be sure to enter a unique name for the task. When the
```
Comment (wblock) [inline userguide/tasks.rst]:
```suggestion
modified to give a more meaningful name to the task. When the
```

File: userguide/tasks.rst [hunk 9]
```diff
@@ -849,109 +849,97 @@ Replication Creation Wizard
...
+source and destination have been configured, click :guilabel:`NEXT` to
```
Comment (wblock) [inline userguide/tasks.rst]:
```suggestion
source and destination have been set, click :guilabel:`NEXT` to
```

File: userguide/tasks.rst [hunk 10]
```diff
@@ -849,109 +849,97 @@ Replication Creation Wizard
...
+configure when the replication runs.
```
Comment (wblock) [inline userguide/tasks.rst]:
```suggestion
choose when the replication will run.
```

File: userguide/tasks.rst [hunk 11]
```diff
@@ -849,109 +849,97 @@ Replication Creation Wizard
...
-A :ref:`Periodic Snapshot Task <Periodic Snapshot Tasks>` is required
-when *PUSH* is selected. Choose a previously-created snapshot task or
-select *Create New* and follow the instructions in
-:ref:`Periodic Snapshot Tasks <zfs_periodic_snapshot_opts_tab>` to
-create a new periodic snapshot schedule.
 
-When the :guilabel:`Direction` is *Pull*, the :guilabel:`Naming Schema`
-of the snapshots to pull from the remote system must be entered.
+The replication task can be configured to run on a schedule or left
+unscheduled and only run when manually activated. Choosing
```
Comment (wblock) [inline userguide/tasks.rst]:
```suggestion
unscheduled and manually activated. Choosing
```

File: userguide/tasks.rst [hunk 12]
```diff
@@ -849,109 +849,97 @@ Replication Creation Wizard
...
-column shows the date and time of the last snapshot taken for a given
-replication task. If the :guilabel:`Last Snapshot` column is not
-shown, it can be enabled from the :guilabel:`COLUMNS` button.
+:menuselection:`Tasks --> Replication Tasks`.
+The task settings that are shown by default can be adjusted by opening
+the :guilabel:`COLUMNS` drop-down. To see more details about the last
+time the replication task ran, click the entry under the
+:guilabel:`State` column. Tasks can also be expanded by clicking
+|ui-chevron-right| for that task. Expanded tasks show all replication
+settings and has |ui-task-run|, |ui-edit|, and |ui-delete| buttons.
```
Comment (wblock) [inline userguide/tasks.rst]:
```suggestion
settings and have |ui-task-run|, |ui-edit|, and |ui-delete| buttons.
```

**Pull request merged**